packages/lib/src/Base.svelte

```diff
@@ -4,18 +4,16 @@
   import { getApi } from './api';
   import initTransaltor from './i18n';
 
-  let {
-    lang
-  }: { lang?: string  } = $props();  
+  let { lang }: { lang?: string } = $props();
   const htmlLang = document.querySelector('html')?.getAttribute('lang');
-  
+
   const apiPromise = getApi();
 
-  const translator = initTransaltor(lang || htmlLang || 'en')
+  const translator = initTransaltor(lang || htmlLang || 'en');
 
   let sesamyDesignTokens = `
     * {
-      --s-main-color: var(--sesamy-main-color, #E71104);
+      --s-main-color: var(--sesamy-main-color, 59 130 246);
```

Comment:
This is more or less the main change. After reading up a bit on tailwind it seems that you need to use rgb colors to get the opacity working correctly. I guess it's something we will need to highlight in the documenation.

Reply:
While this works, it makes it 10 times harder for our clients to use custom CSS variables. This would force them to know how to use `rgb` colors (we provide the `a` in `rbga`).

I'm thinking mabye that's ok since custom CSS variables will be a power-user thing?

@frankjonatanschultz ?

Reply:
I'm not sure that will work since it's in javascript? Tried a bunch of different ways digging out the css variable in javascript but can't get it to work..

Reply:
On our end it will work, IE. passing `#E71104` to our own function.

What won't work is our customers setting `rgb` in their custom CSS variables.

Reply:
Hm im 99% sure we can use hex colors everywhere in tailwind and still set opacity

Reply:
When i try it tailwind seems to automatically convert hex to rgb somehow?

![image](https://github.com/user-attachments/assets/5bffcf57-2f7e-456a-b99b-708aeca0c6e9)

Reply:
`bg-[#FFFFFF50]` works

`bg-[var(--main-color)+50]` doesn't

:(

Reply:
> When i try it tailwind seems to automatically convert hex to rgb somehow?
> 
> ![image](https://private-user-images.githubusercontent.com/15085000/376143594-5bffcf57-2f7e-456a-b99b-708aeca0c6e9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg4OTE4ODAsIm5iZiI6MTcyODg5MTU4MCwicGF0aCI6Ii8xNTA4NTAwMC8zNzYxNDM1OTQtNWJmZmNmNTctMmY3ZS00NTZhLWI5OWItNzA4YWVjYTBjNmU5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE0VDA3Mzk0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdkMjY4MDZmMmRkMmJkMDZlNjMyYTlmYWU5NDA0MjcwNjcwOThiOTg4N2JiMDVkMDJmNWQ4MmVjN2Y3ZWUzZTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.a2k6unwEV9GP6meQgpJHDVu8oPnak1y2c6Q1_cxNHOk)

That's interesting! I wonder if they do compile `hex` to `rgb` themselves? If that's so we should be able to do what Markus did `/0.1` without caring what color code is being used.